client/src/components/layout/BuildsList.js

```diff
@@ -5,25 +5,35 @@ const BuildsList = (props) => {
 
     const [currentBuilds, setCurrentBuilds] = useState([])
 
-    const getBuilds = async () => {
-        try {
-            const response = await fetch("/api/v1/builds/")
-            const parsedResponse = await response.json()
-            setCurrentBuilds(parsedResponse.builds)
-        } catch (error) {
-            console.error(`Error Fetching Build List: ${error}`)
+    const onProfilePage = props.onProfilePage
+    let buildTiles
+    if(onProfilePage){
+        const userBuilds = props.buildsList
+        buildTiles = userBuilds.map((build) => {
+            return <BuildTile key={build.id} build={build} />
+        })
+    } else {        
+        const getBuilds = async () => {
+            try {
+                const response = await fetch("/api/v1/builds/")
+                const parsedResponse = await response.json()
+                setCurrentBuilds(parsedResponse.builds)
+            } catch (error) {
+                console.error(`Error Fetching Build List: ${error}`)
+            }
         }
```

Comment:
hmmm 
- indentation needs fixing 
- Im going to allow for this pattern just for the sake of time, but normally I would ask that you not re-use the BuildList component for the User profile page, because the styling for these on most websites would be dramatically different.